### PR TITLE
Feat/upgrade schema

### PR DESCRIPTION
**Pull request by ratik**
This one is needed to have ability to build Typescript client

https://github.com/neutron-org/neutron-tests/actions/runs/4714905281

### CODE REVIEW

**Review by foxpy — COMMENTED** [review]
Try running `rm -r target/ && find . -iname '*.json' -exec rm {} \;`, and then regenerating schema with `make schema`. At this point, according to `git status`, some files will remain deleted. Let's figure out whether it is a good thing?

**Comment by ratik** [comment]
https://github.com/neutron-org/neutron-tests/actions/runs/4714905281

File: packages/vesting-base/src/msg.rs [hunk 1]
```diff
@@ -108,7 +108,7 @@ pub enum QueryMsg {
     #[returns(VestingState)]
     VestingState {},
     /// Contains messages associated with the managed extension for vesting contracts.
-    #[returns(QueryMsgManaged)]
+    #[returns(Binary)]
```
Comment (NeverHappened) [inline packages/vesting-base/src/msg.rs:111]: why Binary?

Comment (ratik) [reply packages/vesting-base/src/msg.rs:111]: returning Query seems to be a nonsense

**Comment by ratik** [comment]
> Try running `rm -r target/ && find . -iname '*.json' -exec rm {} \;`, and then regenerating schema with `make schema`. At this point, according to `git status`, some files will remain deleted. Let's figure out whether it is a good thing?

`find . -type d -name "schema" | xargs rm -rf`

**Comment by ratik** [comment]
https://github.com/neutron-org/neutron-tests/actions/runs/4717908812

**Review by NeverHappened — APPROVED** [review]

**Review by foxpy — APPROVED** [review]

**Pull request merged**